web/client/components/TOC/fragments/settings/General.jsx

```diff
@@ -46,14 +47,21 @@ class General extends React.Component {
         allowNew: false
     };
 
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
We usually don't use constructor to set initial state.
You can use directly

```js
state = {
  elementFormValue: {}
};

componentDidMount() {
  this.setState({
    elementFormValue: this.props.element
  });
}
```
I didn't test this code so please check if it's working correctly and if componentDidMount is the right lifecycle